apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -77,8 +109,40 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
     sendPythonCodeToMicroBit(pythonCode);
   };
 
+  const onClickSettings = async () => {
+    const results = await dialogControl?.showDialog({
+      type: DialogType.GenericDropdown,
+      title: 'Settings',
```

Comment:
We should translate these strings (could be a follow-up)

Reply:
https://codedotorg.atlassian.net/browse/CT-1167